chain/rosetta-rpc/src/adapters/transactions.rs

```diff
@@ -3,18 +3,21 @@ use std::string::ToString;
 
 use actix::Addr;
 
+use near_account_id::AccountId;
 use near_primitives::hash::CryptoHash;
 use near_primitives::views::SignedTransactionView;
 
-/// A mapping from NEAR transaction or receipt hash to list of receipts.
-/// and a mapping from transaction hashes to transactions.
-/// The latter map is needed to determine the amount of deposit in a single transaction when
-/// converting blocks to Rosetta transactions.
+use crate::models::AccountIdentifier;
+
+/// A mapping from NEAR transaction or receipt hash to list of receipts hashes,
+/// a mapping from transaction hashes to transactions and a mapping of receipts to predecessor_ids.
+/// transactions map is needed to determine the amount of deposit in a single transaction when
+/// converting blocks to Rosetta transactions. Receipts map is reui
```

Comment:
‘Receipts map is reui’? Also, at this point it’s probably easier to move each individual docstring to document the fields with a shorter description of the struct.